stable/aws-for-fluent-bit/templates/daemonset.yaml

```diff
@@ -59,6 +59,19 @@ spec:
             {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+
+        {{- if $.Values.serviceMonitor }}
```

Comment:
@razorsk8jz so the service monitor is the prometheus operator? I don't understand why that flag should also enable/disable the liveness probe. They are separate things right? Fluent Bit can have a health check even if you don't also run prom.

Reply:
I do agree they can be done separately - we could enable a service and the liveness probe without the serviceMonitor - I think it was originally set up this way as it doesnt appear that aws-for-fluent-bit needed a service for anything other than the serviceMonitor.  However we also cant to the healthcheck without the service. 

What would you think about me changing the way this works to have the service always created unreleated to the serviceMonitor and having the healthcheck always on as well?

Reply:
Yea I think that's a good idea, enable the monitoring endpoint by default with healthcheck. Then user chooses if they want to use it (service monitor or/and liveness probe).

Reply:
> Yea I think that's a good idea, enable the monitoring endpoint by default with healthcheck. Then user chooses if they want to use it (service monitor or/and liveness probe).

are you ok with me keeping the liveness probe on by default set to the correct path?

Reply:
Yea.